modules/components/tag/tag.template.html

```diff
@@ -23,7 +23,7 @@
              (keydown.enter)="disableEditMode($event)"
              (keydown.escape)="disableEditMode($event)"
              (click)="editModeActivated ? $event.stopPropagation() : undefined"
-             (blur)="toggleEditMode()">
+             (blur)="onblur($event.target.innerText)">
```

Comment:
can we pass the $event to the method, so that we strip some logic from the template? We may need to stub the typings for $event as Universal will complain